webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -222,8 +222,13 @@ def read_crash_column(crash, column_key):
     return crash.get(column_key, '')
 
 
+@library.filter
+def remove_signature_arguments(signature):
+    return re.sub(r'\(.*\)', '', signature)
```

Comment:
I have vague memories for `.` being greedy, but I don't remember if that's with Python regexes or some other regex system I used over the years. Because of that scar tissue, I would have doing `.*` as `[^\)]+` since we want to match on a bunch of non-right-parens. Your tests pass, so this is fine as is.

Reply:
Wouldn't that fail for nested parentheses? Like `foo:bar(char* baz, int (this ain't valid but whateber))` would end up being `foo:bar)`, since it would consume all the open parenthesis but only one of the closed.

Reply:
Ah, but maybe the current version fails for multiple groups of parentheses? It is a naive way to remove arguments.

Reply:
I don't think I've seen nested parens, but you're right--doing `[^\)]+` doesn't work with nested parens. +1 to keeping what you have now.

Reply:
I did some testing:

```
Python 2.7.13 (default, Jan 19 2017, 14:48:08) 
[GCC 6.3.0 20170118] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import re
>>> p = re.compile(r'\(.*\)')
>>> p.sub('abc', '')
''
>>> p.sub('', 'abc')
'abc'
>>> p.sub('', 'abc(123)')
'abc'
>>> p.sub('', 'abc(123(456))')
'abc'
>>> p.sub('', 'abc(123(456), 333)')
'abc'
```

That seems fine to me for now. If we need something fancier, we can write a parser.